Source/WebCore/accessibility/AccessibilityTableRow.cpp

```diff
@@ -171,5 +173,81 @@ std::optional<unsigned> AccessibilityTableRow::axRowIndex() const
     int value = integralAttribute(aria_rowindexAttr);
     return value >= 1 ? std::optional(value) : std::nullopt;
 }
+
+AccessibilityObject::AccessibilityChildrenVector AccessibilityTableRow::disclosedRows()
+{
+    AccessibilityChildrenVector disclosedRows;
+
+    if (!isARIATreeGridRow())
+        return AccessibilityObject::disclosedRows();
```

Comment:
`AccessibilityChildrenVector disclosedRows;` should go below `if (!isARIATreeGridRow())` since we don't use it if that branch is taken (I know you didn't write this, but maybe we should clean some little things up as part of this transition)